salt/personalised-covers/aws.sls

```diff
@@ -0,0 +1,10 @@
+aws-credentials:
+    file.managed:
+        - name: /home/{{ pillar.elife.deploy_user.username }}/.aws/credentials
+        - source: salt://personalised-covers/config/home-user-.aws-credentials
+        - user: {{ pillar.elife.deploy_user.username }}
```

Comment:
right, not for SQS stuff (because I saw this: https://github.com/elifesciences/personalised-covers/tree/master/src/Queue)
I'll create a user for this project and commit the credentials on the builder-private repository, where the pillars for EC2 instances come from